docker/dyno/entrypoint.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 echo "Staring server.."
-gunicorn -b 0.0.0.0 dyno.app:app --capture-output  -t 90 -w 8
+gunicorn -b 0.0.0.0 dyno.app:app --capture-output  -t 90 -w 8 --reload
```

Comment:
For same reason explained above, it allows hot-reloading the static code while editing in dev mode.

Reply:
👍